resources/src/event_handlers/message_event_handler.rs

```diff
@@ -23,6 +23,12 @@ pub async fn handle_message_event(body: &Value) {
         .as_str()
         .unwrap_or("invalid_enterprise_user_id");
     let _user: &str = body["event"]["user"].as_str().unwrap_or("invalid_user");
+    let is_development = get_env_var("IS_DEVELOPMENT");
+    // Stop the function if this is a development environment and outside the test channel
+    if channel != "C0351GJ62Q0" && is_development == "true" {
```

Comment:
Parse the `is_development` flag and use it as a regular boolean. Don't use string comparison here.

Something like:
```
let is_development: bool = env_value.parse().unwrap();
```

Also you might want to do this once in main and pass it into the handlers.